.github/workflows/helm-chart-release.yml

```diff
@@ -56,3 +59,14 @@ jobs:
           charts_dir: charts
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Prepare GH pages readme
+        run: |
+          mkdir -p ./to-gh-pages
+          cp -f charts/kubewarden-controller/README.md ./to-gh-pages
```

Comment:
Are you sure you don't have to checkout the gh-pages branch that has just been updated by the helm chart releaser?

Reply:
True! with https://github.com/kubewarden/helm-charts/pull/37/commits/0a5b2b507cc925cdbebf65356cfce4ade5f0bb92 it should never happen. Checked on my fork, see the file `foo` on the gh-pages branch: https://github.com/viccuad/helm-charts/tree/gh-pages